content/styles/00-base/_variables.css

```diff
@@ -1,11 +1,17 @@
-/* custom properties */
 :root {
-
-  /* Colors */
-  --greenLink: #279e6a;
-  --greenLinkHover: #2cbe4e;
-
-  --blackFont: #000;
-
-  --greyBg: #fafafa;
+  --primary-color: #3dd07d;
+  --secondary-color: #4e8;
+    
+  --primary-bg-color: #fefefe;
+  --secondary-bg-color: #fafafa;
+  
+  --black: #000;
+  --gray: #333;
+  --white: #fff;
+  
+  --gray-opacity: rgba(50, 50, 50, 0.15);
+  
+  --head-font: 'Poppins', Arial, sans-serif;
```

Comment:
`--title-font`?

Reply:
However I feel like it will be applied once to `h1`>`h6` and that's it.
In which case it would be used in 1 place => no need of a var.

Reply:
true and false, I use it also in the styleguide presentation